integration/emr/alluxio-emr.sh

```diff
@@ -244,17 +249,16 @@ USAGE_END
   local restore_from_backup_uri=""
   local files_list=""
   local nvme_capacity_usage=""
-
-  if [[ "$#" -lt "1" ]]; then
-    echo -e "No root UFS URI provided"
-    print_help 1
-  fi
-
-  local root_ufs_uri="${1}"
-  shift
-  while getopts "b:d:f:i:n:p:s:" option; do
-    OPTARG=$(echo -e "${OPTARG}" | tr -d '[:space:]')
+  local root_ufs_uri=""
+  local client_only="false"
+  while getopts ":u:b:d:f:i:n:p:s:c" option; do
```

Comment:
the getopts string should be `"b:cd:f:i:n:p:s:u:"`
`:` means expect an argument value afterwards, so `c` doesn't get one